deps/Makefile

```diff
@@ -671,7 +677,10 @@ else ifeq ($(LLVM_VER),3.7.1)
 $(eval $(call LLVM_PATCH,llvm-3.7.1))
 $(eval $(call LLVM_PATCH,llvm-3.7.1_2))
 $(eval $(call LLVM_PATCH,llvm-3.7.1_3))
+$(eval $(call LLVM_PATCH0,llvm-D14260))
 $(LLVM_SRC_DIR)/llvm-3.7.1_2.patch-applied: $(LLVM_SRC_DIR)/llvm-3.7.1.patch-applied
+$(LLVM_SRC_DIR)/llvm-3.7.1_3.patch-applied: $(LLVM_SRC_DIR)/llvm-3.7.1_2.patch-applied
+$(LLVM_SRC_DIR)/llvm-D14260.patch-applied: $(LLVM_SRC_DIR)/llvm-3.7.1_3.patch-applied
```

Comment:
I find it tedious to look through all existing patches, extracting all file names, and comparing against all file names in the new patch, manually determining which patches conflict. It's easy to make a mistake. Applying patches is surely so fast that the missed Makefile parallelization doesn't matter. I think the only reason the patches aren't applied serially is that this is difficult to do in Makefiles, hence the dependencies.

Please advise -- I'm not objecting, I'm just giving the reasons for why I did things the way I did.